test/brod_group_coordinator_SUITE.erl

```diff
@@ -0,0 +1,127 @@
+%%%
+%%%   Copyright (c) 2015-2018 Klarna Bank AB (publ)
+%%%
+%%%   Licensed under the Apache License, Version 2.0 (the "License");
+%%%   you may not use this file except in compliance with the License.
+%%%   You may obtain a copy of the License at
+%%%
+%%%       http://www.apache.org/licenses/LICENSE-2.0
+%%%
+%%%   Unless required by applicable law or agreed to in writing, software
+%%%   distributed under the License is distributed on an "AS IS" BASIS,
+%%%   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%%%   See the License for the specific language governing permissions and
+%%%   limitations under the License.
+%%%
+
+%% @private
+-module(brod_group_coordinator_SUITE).
+-define(CLIENT_ID, ?MODULE).
+-define(OTHER_CLIENT_ID, other_coordinator_id).
+-define(TOPIC, <<"brod-group-subscriber-1">>).
```

Comment:
This topic name collides with the `group_subscriber_SUITE`, please rename to something unique